jpf-symbc/bin/jpf-sv-comp.sh

```diff
@@ -46,7 +46,7 @@ trap "rm -rf $DIR" EXIT
 mkdir -p $DIR/target/classes
 
 # build src files from benchmark
-/usr/lib/jvm/java-8-openjdk-amd64/bin/javac -g -cp $DIR/target/classes:../wit4java/sv-benchmarks/java/common:../wit4java/sv-benchmarks/java/securibench/micro:../wit4java/sv-benchmarks/java/java-ranger-regression/infusion/impl -d $DIR/target/classes "${BM[@]}"
+/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home/bin/javac -g -cp $DIR/target/classes:../wit4java/sv-benchmarks/java/common:../wit4java/sv-benchmarks/java/securibench/micro:../wit4java/sv-benchmarks/java/java-ranger-regression/infusion/impl -d $DIR/target/classes "${BM[@]}"
```

Comment:
why have you changed this path?

Reply:
Oh it's my mistake. I changed this to test on my machine..I fixed this. Thank you!